distribution/changelog.txt

```diff
@@ -6,6 +6,7 @@
 - Feature: [#14538] [Plugin] Add property for getting current plugin api version.
 - Feature: [#14620] [Plugin] Add properties related to guest generation.
 - Feature: [#14636] [Plugin] Add properties related to climate and weather.
+- Feature: [#14731] Opaque water (like in RCT1).
```

Comment:
Should this have reflected the "Transparent water" string?

Reply:
Nah, don’t think so. Transparent water is what we already had, opaque is what we added.